### DISCUSSION TITLE
PDF文件预览失败

### DISCUSSION THREAD

**Discussion by xtxt19931207** [AlistGo/alist#4443] [Q&A]
载入 PDF 时发生错误
PDF.js v3.0.44 (build: 6852001b0)
信息：Failed to fetch

在PC的edge浏览器和安卓的edge浏览器都无法预览，安卓手机自带浏览器（小米浏览器）可正常预览。

**Comment by talentestors** [comment]
> 载入 PDF 时发生错误 PDF.js v3.0.44 (build: 6852001b0) 信息：Failed to fetch
> 
> 在PC的edge浏览器和安卓的edge浏览器都无法预览，安卓手机自带浏览器（小米浏览器）可正常预览。

看看你是不是这种情况：https://www.bilibili.com/read/cv21387408/

**Reply by xtxt19931207** [reply]
第一时间在网上搜索过各种解决办法，也尝试过禁用IDM，都没有效果，最终还是没有解决

**Comment by tmqxiujin** [comment]
请问解决了吗，我的也出现了这个问题，但是我的在手机浏览器上可以预览，在某一台电脑上关闭IDM自动下载也可以预览，但是在其他另外两台电脑上试的无法预览

**Reply by LONGZR007** [reply]
我F12看说是不能用http，我改https好了

**Reply by lzgverynice** [reply]
> 我F12看说是不能用http，我改https好了

没有买域名的怎么改？

**Comment by LONGZR007** [comment]
我F12看说是不能用http，我改https好了

**Comment by artiark** [comment]
@LONGZR007 F12也说不能用http，求怎么改

**Comment by huyiwei** [comment]
自己编了一下源码，使用浏览器直接预览了 pc:浏览器 前缀是pc:只有pc打开会显示，mobile:只有手机打开会显示。并且会记录上次的打开方式，一下子完美了~
附上一个可用的 "alicdn.js":"http://g.alicdn.com/teambition/pdfviewer/1.2.0/index.html?file=$e_url"
![image](https://github.com/user-attachments/assets/5b3eff6b-a217-475f-94b1-174c71142120)

![image](https://github.com/user-attachments/assets/473b0a4c-ead5-4135-b525-2059a3fef240)

**Reply by wuyue2011** [reply]
这个放到Alist哪个目录下

**Comment by yowaimono** [comment]
我也遇到这个问题了，并且只有访问夸客网盘才会有这个问题，阿里云盘等都是正常的

**Reply by kuiyr0810** [reply]
> 我也遇到这个问题了，并且只有访问夸客网盘才会有这个问题，阿里云盘等都是正常的

是的，只有夸克

**Comment by qcnhy** [comment]
![image](https://github.com/user-attachments/assets/683ac19f-b589-434d-8abe-f87059adb398)
同样 夸克
